src/apiv2.ts

```diff
@@ -326,12 +326,8 @@ export class Client {
       compress: options.compress,
     };
 
-    if (this.opts.proxy) {
```

Comment:
I imagine if `opts.proxy` isn't used any longer, do we need to drop it from the options in the types?

Reply:
Good point - done